packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -331,7 +335,19 @@ abstract class ImplicitlyAnimatedWidgetState<T extends ImplicitlyAnimatedWidget>
   /// 2. Take the value returned from the callback, and store it. This is the
   /// value to use as the current value the next time that the forEachTween()
   /// method is called.
+  ///
+  /// Subclasses that contain properties based on tweens created by
+  /// forEachTween() should override onTweensUpdated() to update those
```

Comment:
nit: here and elsewhere, in the docs, rather than `foo()`, use `[foo]`.

Reply:
Should I change the existing references that weren't using "[]"?

Reply:
yes please!